spinalcordtoolbox/utils.py

```diff
@@ -138,83 +156,19 @@ def is_exe(fpath):
     return None
 
 
-def __get_branch():
+def extract_fname(fpath):
```

Comment:
I would just make this

```suggestion
def splitext(filename):
```

If users need the folder path, they already have `os.path.split()`. `sct.utils.splitext()` can be a drop in neuroimaging-aware replacement for `os.path.splitext()`.

Reply:
great suggestion. fixed in ea0c10616049b8a433c2e7ebba5aedf3e498568b